### PR TITLE
Includes Debugs, PartiaLEqs and FMT

### PR DESCRIPTION
**Pull request by naomijub**

### CODE REVIEW

File: src/docker.rs [hunk 1]
```diff
@@ -25,6 +25,7 @@ pub struct DockerClient {
 /// * `PUT` - Write a version of a document
 /// * `Delete` - Deletes the specific document at a given valid time
 /// * `Evict` - Evicts a document entirely, including all historical versions (receives only the ID to evict)
+#[derive(Debug, PartialEq)]
 pub enum Action {
```
Comment (naomijub) [inline src/docker.rs:29]: Added Debug and PartialEq for testing purposes

File: src/types/query.rs [hunk 2]
```diff
@@ -10,7 +10,7 @@ use std::convert::TryFrom;
 /// * `order_by` is responsible for defining the order in which the response will be represented, **optional**. Argument is a vector with strings containing the element and how to order (`:asc` or `:desc`) `vec!["time :desc", "device-id :asc"]`.
 /// * `limit` is responsible for defining the limit size of the response, **optional**. Argument is a usize.
 /// * `offset` is responsible for defining the offset of the response, **optional**. Argument is a usize.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct Query {
```
Comment (naomijub) [inline src/types/query.rs:14]: Added Debug for testing purposes

File: src/docker.rs (hunk 1 above)
Comment (otaviopace) [reply src/docker.rs:29]: But are you going to remove it?

File: src/types/query.rs (hunk 2 above)
Comment (otaviopace) [reply src/types/query.rs:14]: But are you going to remove it?

File: src/docker.rs (hunk 1 above)
Comment (otaviopace) [reply src/docker.rs:29]: Or is it a permanent thing to help the crate users?

File: src/types/query.rs (hunk 2 above)
Comment (otaviopace) [reply src/types/query.rs:14]: Or is it a permanent thing to help the crate users?

File: src/docker.rs (hunk 1 above)
Comment (naomijub) [reply src/docker.rs:29]: When using with ATM-Crux I noticed that this reduced testability.
What do you think?

File: src/types/query.rs (hunk 2 above)
Comment (naomijub) [reply src/types/query.rs:14]: When using with ATM-Crux I noticed that this reduced testability.
What do you think?

**Comment by naomijub** [comment]
@otaviopace do you think we can consider this a `1.0.0`?

**Pull request merged**